sql/updates/world/3.3.5/2016_05_15_15_world.sql

```diff
@@ -0,0 +1,52 @@
+-- Bunthen Plainswind (NPC 11798) and Silva Fil'naveth (NPC 11800), Moonglade Druid Flight Masters
+UPDATE `creature_template` SET `AIName`= 'SmartAI', `ScriptName`= '' WHERE `entry` IN (11798,11800);
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id` IN (4041,4042);
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(4041,0,0,"I'd like to fly to Rut'theran Village.",                             7573,1,1,   0,0,0,0,'',0),
+(4041,1,0,"Do you know where I may find the Half Pendant of Aquatic Agility?",  8035,1,1,4225,0,0,0,'',0),
+(4041,2,0,"Do you know where I may find the Half Pendant of Aquatic Agility?",  8035,1,1,4226,0,0,0,'',0),
+(4042,0,0,"I'd like to fly to Thunder Bluff.",                                 12804,1,1,   0,0,0,0,'',0),
+(4042,1,0,"Do you know where I may find the Half Pendant of Aquatic Endurance?",8036,1,1,4223,0,0,0,'',0),
+(4042,2,0,"Do you know where I may find the Half Pendant of Aquatic Endurance?",8036,1,1,4224,0,0,0,'',0);
+
+-- Link gossip_menu_option.menu_id (gossip_menu.entry) to npc_text.id (gossip_menu.text_id)
+DELETE FROM `gossip_menu` WHERE `text_id` IN (4917,4918,5373,5374,5375,5376);
+INSERT INTO `gossip_menu` (`entry`,`text_id`) VALUES
+(4042,4917),
+(4042,4918),
+(4223,5373),
+(4224,5374),
+(4225,5375),
+(4226,5376);
```

Comment:
`gossip_menu`.`entry` & `gossip_menu_option`.`menu_id` 4041+4042 are already in the DB, but I picked 4223-4226 from available entries in the `gossip_menu` table because those numbers are very close to other similar entries in use. I could not locate the values in the general 3.3.0 sniff sample I got from Aokromes a couple of months ago. Since these particular `gossip_menu` entries got removed in Cata (patch 4.0.3a), they might be hard to find. The `text_id` values are valid `npc_text`.`ID` gossip texts.